homeassistant/components/flux_led/light.py

```diff
@@ -271,23 +270,22 @@ def __init__(
         """Initialize the light."""
         super().__init__(coordinator)
         self._bulb: WifiLedBulb = coordinator.device
-        self._name = name
-        self._unique_id = unique_id
+        self._attr_name = name
+        self._attr_unique_id = unique_id
         self._ip_address = coordinator.host
         self._mode = mode
         self._custom_effect_colors = custom_effect_colors
         self._custom_effect_speed_pct = custom_effect_speed_pct
         self._custom_effect_transition = custom_effect_transition
-
-    @property
-    def unique_id(self) -> str | None:
-        """Return the unique ID of the light."""
-        return self._unique_id
-
-    @property
-    def name(self) -> str:
-        """Return the name of the device."""
-        return self._name
+        old_protocol = self._bulb.protocol == "LEDENET_ORIGINAL"
+        if self.unique_id:
+            self._attr_device_info = {
+                "connections": {(dr.CONNECTION_NETWORK_MAC, self.unique_id)},
+                ATTR_MODEL: f"0x{self._bulb.raw_state[1]:02X}",
+                ATTR_SW_VERSION: "1" if old_protocol else str(self._bulb.raw_state[10]),
```

Comment:
We will be able to get rid of the positional args once https://github.com/Danielhiversen/flux_led/pull/96 is published.

But I don't want to hold up the next PR on top of this that will actually fix the RGBW/RGBWW/Color temp lights so I'll do another turn of this after we bump the upstream lib.